doc/src/site/sphinx/configuration.rst

```diff
@@ -0,0 +1,307 @@
++--------------------------------+--------------------------------------------------------------------+
+| spark.cores.max                | the maximum amount of CPU cores to request for the application     |
+|                                | from across the cluster (not from each machine).                   |
++--------------------------------+--------------------------------------------------------------------+
+| spark.mesos.extra.cores        | set the extra amount of cpus to request per task. This setting is  |
+|                                | only used for Mesos coarse grain mode. The total amount of cores   |
+|                                | requested per task is the number of cores in the offer plus the    |
+|                                | extra cores configured. Note that total amount of cores the        |
+|                                | executor will request in total will not exceed the spark.cores.max |
++--------------------------------+--------------------------------------------------------------------+
+| spark.mesos.coarse             | Set the run mode for Spark on Mesos.                               |
++--------------------------------+--------------------------------------------------------------------+
+| spark.executor.memory          | Amount of memory to use per executor process (e.g. 2g, 8g).        |
++--------------------------------+--------------------------------------------------------------------+
+| spark.driver.cores             | Number of cores to use for the driver  only in cluster mode.       |
++--------------------------------+--------------------------------------------------------------------+
+| spark.driver.memory            | Amount of memory to use for the driver process.                    |
++--------------------------------+--------------------------------------------------------------------+
+
+(**) Note: Spark on Mesos also supports cluster mode, where the driver is launched in the cluster and the client can
+find the results of the driver from the Mesos Web UI. To use cluster mode, you must start the MesosClusterDispatcher
+in your cluster via the sbin/start-mesos-dispatcher.sh script, passing in the Mesos master url (e.g: mesos://host:5050).
+
+From the client, you can submit a job to Mesos cluster by running spark-submit and specifying the master url to the url
+of the MesosClusterDispatcher (e.g: mesos://dispatcher:7077). You can view driver statuses on the Spark cluster Web UI.
+
+.. _cluster-config-yarn:
+
+Sparkta + Apache YARN
+---------------------
+
+You must have correctly configured and deployed Apache Yarn. You can obtain information about how to do it in
+|yarn_doc|.
+
+.. |yarn_doc| raw:: html
+
+   <a href="https://hadoop.apache.org/docs/current/hadoop-yarn/hadoop-yarn-site/YARN.html"
+   target="_blank">here</a>
+
+The next step is set up Sparkta's YARN configuration:
```

Comment:
Please remove from this line to the bottom. This doc goes to master branch so it shouldn't be uncompleted